pyproject.toml

```diff
@@ -54,7 +54,7 @@ dependencies = [
     "sendgrid>=6.4.8,<6.5.0",
     "sentry-sdk~=2.0",
     "slack_sdk>=3.10,<4.0",
-    "sretoolbox~=2.6",
+    "sretoolbox==2.7",
```

Comment:
I needed to go to [PEP 440](https://peps.python.org/pep-0440) to check that `X.Y` and `X.Y.0` is the same 😛 

Where did the "explicit is better than implicit" from PEP20 go? 😂

Reply:
changed in https://github.com/app-sre/qontract-reconcile/pull/5224/commits/a08a3ee42dd536d4b0aa45dea6b9816dded581a8